target/checkstyle-cachefile

```diff
@@ -0,0 +1,4 @@
+#Mon Aug 24 20:28:14 CST 2020
+module-resource*?\:config/sevntu_suppressions.xml=F0847992EF5169789F9CCDE1E0312EBA8C027703
+module-resource*?\:show.patch=5AB37DF7A2B1C28DA872C57C26358B2A8C4DA61C
+configuration*?=52B10C8F03C3D56953783503957FB08EAF5DB67B
```

Comment:
add `target` folder to `.gitignore` 
such file should not be in GIT

Reply:
done